### PR TITLE
testssl.sh: init at 2.9.5-1

### PR DESCRIPTION
**Pull request by etu**
###### Motivation for this change
Add testssl.sh which is a nice utility for testing TLS/SSL capabilities of servers without having to use any kind of web-service. It's very useful for testing setups of services before deployment and such.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/networking/testssl/default.nix
Comment (vyp) [inline pkgs/applications/networking/testssl/default.nix]: `rev = "v${version}";` works.

File: pkgs/applications/networking/testssl/default.nix
Comment (vyp) [inline pkgs/applications/networking/testssl/default.nix]: A maintainer must be set, did you want to maintain this package?

File: pkgs/applications/networking/testssl/default.nix
Comment (etu) [reply pkgs/applications/networking/testssl/default.nix]: Oh, ok. Changing that :)

File: pkgs/applications/networking/testssl/default.nix
Comment (etu) [reply pkgs/applications/networking/testssl/default.nix]: Sure, I can do that. I will amend and push that together with the version fix.

File: pkgs/applications/networking/testssl/default.nix
Comment (joachifm) [inline pkgs/applications/networking/testssl/default.nix]: Please do not pass all of `pkgs`, instead declare each parameter you need directly. Also note that `lib` is typically accessed via `stdenv.lib`.

File: pkgs/applications/networking/testssl/default.nix
Comment (joachifm) [inline pkgs/applications/networking/testssl/default.nix]: Nit: `makeWrapper` is build-time only and could be listed in `nativeBuildInputs` instead.

File: pkgs/applications/networking/testssl/default.nix
Comment (joachifm) [inline pkgs/applications/networking/testssl/default.nix]: Please drop the `"foo is a ..."` from the short description.

File: pkgs/applications/networking/testssl/default.nix
Comment (joachifm) [inline pkgs/applications/networking/testssl/default.nix]: Nit: this comment and those below just restate what the code immediately following obviously does, so seem redundant (similar to `int x = 1+1; /* add 1 to 1 */`).

**Review by joachifm — CHANGES_REQUESTED** [review]
Thank you. Added a few nits for you to consider, nothing major but I think they ought be addressed before merging.

File: pkgs/applications/networking/testssl/default.nix
Comment (joachifm) [inline pkgs/applications/networking/testssl/default.nix]: Seeing as this is a bash script, you could avoid the wrapper indirection by simply injecting those values into the script itself.

Comment (etu) [reply pkgs/applications/networking/testssl/default.nix]: I'll fix this and the notes above in an amend :)

**Review by joachifm — APPROVED** [review]

**Pull request merged**
